### DISCUSSION TITLE
Open API - astrological data via HTTP

### DISCUSSION THREAD

**Discussion by sengiv** [gen-so/Genso.Astrology#1] [Announcements]
We are excited to announce the availability of our free **Open API** service.
Now any developer can access **accurate astrological data** via a simple `HTTP GET` request.
This service is new & much is still under development.

Please let us know, what kind of data you'd like to access via this API in the future.
We are open to any suggestions & feel free to ask if you have challenges using this API.
Visit the [API documentation page](https://www.vedastro.org/API/) for the latest list of available calls. 

Open API #20

*[Function where the magic happens](https://github.com/VedAstro/VedAstro/blob/483c2b07520417aaac76811891679c4cbdf30fb2/API/OpenAPI.cs#L36)

**Comment by sengiv** [comment]
## Example API call for 7 types of planet strength
![image](https://user-images.githubusercontent.com/43817262/226313171-0d4f8325-d0b6-4833-a6b2-f5b4d3e37788.png)

thanks to user @kristaps.bunts for suggesting this 👊 🙏

**Comment by zoneh5** [comment]
Дізнатись більше про своє життєве призначення допомагає калькулятор долі, який аналізує дату народження для визначення ключових рис характеру та можливостей. Завдяки [калькулятор долі](https://matrixdestiny.net/uk/) ви можете отримати глибше розуміння власного шляху та приймати обґрунтовані рішення.
